test/test_unstructured_2d.jl

```diff
@@ -90,6 +90,27 @@ isdir(outdir) && rm(outdir, recursive=true)
               0.00013645610312810813],
       tspan = (0.0, 0.5))
   end
+
+  @trixi_testset "elixir_shallow_water_ec.jl" begin
```

Comment:
```suggestion
  @trixi_testset "elixir_shallowwater_ec.jl" begin
```
To be consistent with, e.g., `hypdiff`, `eulermulti` etc., shouldn't the abbreviated name for the SWE be just `shallowwater`?

Reply:
Sure, I can remove the extra underscore